kubernetes/cosmos/templates/pfsd-deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
               containerPort: 9992
               protocol: TCP
           env:
+            - name: PFSD_READONLY
+              value: {{ .Values.persistentVolume.readOnly }}
```

Comment:
This should be quoted or `| quote`. I just tried it with true and got a templating error.

Reply:
Done!